File: csas-demo-be/src/main/java/cz/csas/demo/Pagination.java
package cz.csas.demo;

import lombok.Builder;
import lombok.Value;

/**
 * Informace o strankovani a poctu polozek
 */
@Value
@Builder
public class Pagination {
    /** Aktualni stranka */
    int pageNumber;
    /** Celkovy pocet stranek */
    int pageCount;
    /** Velikost stranky */
    int pageSize;
    /** Celkovy pocet polozek */
    int totalItemCount;
}

File: csas-demo-fe/src/app/models/pagination.ts
/** Informace o strankovani */
export interface Pagination {
  /** Aktualni stranka */
  pageNumber: number;
  /** Celkovy pocet stranek */
  pageCount: number;
  /** Velikost stranky */
  pageSize: number;
  /** Celkovy pocet prvku */
  totalItemCount: number;
}
